Samples/iOS-Swift/iOS-Swift/ErrorsViewController.swift

```diff
@@ -13,6 +13,13 @@ class ErrorsViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         addDSNDisplay(self, vcview: dsnView)
+
+        if ProcessInfo.processInfo.arguments.contains("--io.sentry.ui-test.use-custom-feedback-button") {
```

Comment:
`m`: Would it make sense moving this live to `SentrySDKOverrides.swift`?

Reply:
Yep!